### PR TITLE
Upgrade Ember to 3.3, and simplify the benchmark

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
I switched the commands to local versions so it's easier to know what it being used.

### CODE REVIEW

File: frameworks/keyed/ember/app/utils/benchmark-helpers.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const _random = (max) => {
```
Comment (NullVoxPopuli) [inline frameworks/keyed/ember/app/utils/benchmark-helpers.js]: I pulled this from the react folder

File: frameworks/keyed/ember/app/components/my-table.js [hunk 2]
```diff
@@ -1,35 +1,101 @@
...
+    });
+  }
+
+  @action async swapRows() {
+    startMeasure('swapRows');
+
+    this.set('data', swapRows(this.data));
+  }
+
+  @action async remove(id) {
```
Comment (rtablada) [inline frameworks/keyed/ember/app/components/my-table.js]: Probably don't use `async` without `await` or promises.

Comment (NullVoxPopuli) [reply frameworks/keyed/ember/app/components/my-table.js]: the intent was to allow the UI to not be blocked due to the long running time of the benchmark helpers

Comment (leeoniya) [reply frameworks/keyed/ember/app/components/my-table.js]: startMeasure and stopMeasure can be dropped completely, btw. they are not used by this benchmark.

Comment (krausest) [reply frameworks/keyed/ember/app/components/my-table.js]: The real measurement happens with the test driver in webdriver-ts that uses chrome's timeline feature to measure the duration. Anything else is just an approximation (sometimes not too bad to validate the measurements).

Comment (NullVoxPopuli) [reply frameworks/keyed/ember/app/components/my-table.js]: oh neat. ok, yeah, I'll remove all the measuring stuff.

Comment (NullVoxPopuli) [reply frameworks/keyed/ember/app/components/my-table.js]: @krausest done

**Comment by krausest** [comment]
Thanks all. I allowed myself to remove yarn (so far my experience wasn't good: #221 and #182). I suppose it's fine that way too, isn't it?

Sadly it seems like 3.3 with the changes is a small step backwards regarding performance:
bildschirmfoto 2018-08-26 um 20 56 17
Did you also see a performance regression?

**Comment by NullVoxPopuli** [comment]
Depends, honestly. I can tons of tests, and was comparing vs react instead of vanilla. Also had to make sure I used private browsing with no extensions. And not have much running on my computer.

There were some tests that had the updates much faster than react. Some that the updates were pretty much the same. And some that were slightly slower.

Lots of variance. :/

**Pull request merged**